concepticondata/references/references.bib

```diff
@@ -3121,4 +3121,18 @@ @Book{Hartmann2013
     address = {Leipzig},
     editor = {Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley},
     year = {2013}
-}    
+}
+
+@misc{Lundgren2020,
```

Comment:
instead of misc, we use "thesis" or "mastersthesis" in bibtex.

See the bibtex I provided [here](https://github.com/lexibank/lexibank/issues/208).

Reply:
and sorry for not mentioning this, as this would have saved some work.